src/routes/profile.js

```diff
@@ -323,7 +323,11 @@ router.post('/:profileUsername', checkProfileOwnership, (req, res) => {
   // console.log(req.body.nationality);
   // console.log(req.body.nationCode);
   // console.log(req.body.hideStats);
+  // console.log(req.body.pronoun);
 
+  if(!["he/him","she/her","they/them"].includes(req.body.pronoun)){
```

Comment:
Looks great!
Could you space out the commas here please?

`"he/him", "she/her", "they/them"` vs `"he/him","she/her","they/them"`